.bazelci/presubmit.yml

```diff
@@ -1,24 +1,34 @@
 ---
 default_targets: &default_targets
+- "--" # Allows negative patterns; hack for https://github.com/bazelbuild/continuous-integration/pull/245
 - "..."
 - "@docs//..."
 - "@examples//..."
+# Bindgen currently only has a working toolchain for 18.04
+- "-@examples//ffi/rust_calling_c/simple/..."
 platforms:
   ubuntu1404:
     build_targets: *default_targets
     test_targets: *default_targets
   ubuntu1604:
     build_targets: *default_targets
     test_targets: *default_targets
+  ubuntu1804:
+    build_targets: *default_targets
+    test_targets:
+      - "..."
+      - "@docs//..."
+      - "@examples//..."
   macos:
     osx_targets: &osx_targets
-    - "--" # Allows negative patterns; hack for https://github.com/bazelbuild/continuous-integration/pull/245
+    - "--"
```

Comment:
Why remove the comment?

Reply:
I moved the comment up to the first occurence